WeakAuras/AuraWarnings.lua

```diff
@@ -42,18 +42,21 @@ end
 
 local severityLevel = {
   info = 0,
-  warning = 1,
-  error = 2
+  sound = 1,
+  warning = 2,
+  error = 3
 }
 
 local icons = {
   info = [[Interface/friendsframe/informationicon.blp]],
+  sound = [[interface/common/voicechat-on.blp]],
```

Comment:
i dislike this icon, one i like is an atlas

`sound = [[chatframe-button-icon-voicechat]],`

in AddMessages
```
      if C_Texture.GetAtlasInfo(icon) then
        result = result .. "|A" .. icon .. ":12:12:0:0:64:64:4:60:4:60|t"
      else
        result = result .. "|T" .. icon .. ":12:12:0:0:64:64:4:60:4:60|t"
      end
```
in AceGUIWidget-WeakAurasDisplayButton.lua/UpdateWarning L1371
```
      if C_Texture.GetAtlasInfo(icon) then
        self.warning:SetNormalAtlas(icon)
      else
        self.warning:SetNormalTexture(icon)
      end
```